packages/flutter/lib/src/widgets/form.dart

```diff
@@ -256,6 +257,9 @@ class FormField<T> extends StatefulWidget {
   /// autovalidates, this value will be ignored.
   final bool autovalidate;
 
+  /// If true, indicates that the widget is enabled
```

Comment:
please end sentences with a period. Also, maybe add a second paragraph explaining the implications (e.g. does this prevent interaction, prevent validation, etc)